doc/08_dev_talks/paf24/student_roles24.md

```diff
@@ -0,0 +1,130 @@
+# Student Roles for Autonomous Vehicle Development Team
+
+## Role overview
+
+2-3 Students per Role
+
+- **Systems Engineer**
+  - Oversee the entire development process, ensuring smooth interaction between different subsystems (perception, planning, control, decision-making, etc.).
+  - Define system-level architecture, ensuring each module (e.g., sensors, planning, control) interacts through well-defined interfaces.
+  - Manage requirements (e.g. in issues) and ensure each team's outputs align with the overall system goals, including performance, reliability, and safety standards.
+  - Serve as the point of contact for inter-team communication, ensuring alignment between roles such as Perception Engineers, Control Engineers, and Decision-Making Engineers.
+  - Develop and enforce a systems integration strategy that covers continuous testing, validation, and verification of the autonomous driving stack.
+  - Ensure proper data flow between modules using middleware (e.g., ROS).
+  - Define and monitor key performance indicators (KPIs) for each subsystem, ensuring they collectively meet reliability, stability, and safety goals.
+  - Provide leadership in prioritizing tasks, resource allocation, and responsibility distribution, ensuring the team meets project milestones.
+  - Guide the team in prioritizing tasks and responsibilities, ensuring timely progress toward milestones.
+  - Create detailed, version-controlled documentation of system design, module interactions, integration protocols, and architectural decisions, ensuring scalability and ease of future development.
+  - Lead risk management efforts, identifying system-level risks and developing mitigation strategies.
+  - Focus on the system’s overall functionality, making sure that all subsystems work harmoniously to create a reliable, autonomous vehicle capable of safe driving in the CARLA simulation.
+- **Decision-Making Engineer**
+  - Develop the vehicle’s decision-making logic for dynamic driving scenarios (e.g., merging lanes, overtaking, yielding at intersections).
+  - Implement high-level decision-making algorithms (e.g., rule-based systems, behavior trees, or reinforcement learning) to choose the best action at any given time.
+  - Ensure the vehicle follows traffic laws, responds to signals and signs, and interacts safely with other vehicles and pedestrians.
+  - Design algorithms to handle edge cases, such as sudden obstacles, unpredictable pedestrian behavior, construction sides or vehicle breakdowns.
+  - Collaborate with perception, planning, and control engineers to ensure the decision-making module aligns with the data and actions generated by other subsystems.
+  - Simulate and validate decision-making in various complex driving scenarios within CARLA, such as navigating congested traffic or adverse weather conditions.
+  - Ensure decision-making algorithms are interpretable and explainable to enhance debugging and safety validation.
+- **Machine Learning Engineer**
```

Comment:
Machine Learning Engineer --> Ich bin mir nicht sicher, ob man den braucht oder ob das nicht eine Kompetenz der Engineers in den verschiedenen Bereichen sein sollte.